source/glue/amc_transformations.py

```diff
@@ -104,6 +92,8 @@
         print("ERROR: Invalid user-defined value for dataset period:")
         print(user_defined_partition_size)
         exit(1)
+if 'country_code' in args:
+    country_code = args['country_code']
```

Comment:
else:
    print("Missing required arg: country_code")

Reply:
fixed: [d106f81](https://github.com/aws-solutions/amazon-marketing-cloud-uploader-from-aws/pull/83/commits/d106f81cd66d51c36a5504dc57c793cd0468abdd) and [78d9514](https://github.com/aws-solutions/amazon-marketing-cloud-uploader-from-aws/pull/83/commits/78d95149ca8b173c597bb89562884700bafae0f8)